app/controllers/kaui/accounts_controller.rb

```diff
@@ -54,6 +54,17 @@ def new
   end
 
   def create
+
+    test_phone = KauiPhoneValid.new
```

Comment:
Shouldn't the validation be in the `Kaui::Account` model instead? That way, it's easily extensible to other fields.

Reply:
Done